app/views/vm_common/_evacuate.html.haml

```diff
@@ -0,0 +1,53 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "vmCloudEvacuateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Evacuate Host')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto-select Host?')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "auto_select_host",
+               'ng-model'    => "vmCloudModel.auto_select_host",
+               :miqrequired  => true,
+               :checkchange  => true,
+               'ng-disabled' => "hosts.length == 0"}
```

Comment:
This shouldn't be disabled, since disabled elements don't get submitted in the form, and we need this param to be processed as if the user had selected it, even when there's no reasonable choice not to select. Alternatively, we'd need to check in the form submission logic and consider auto_select_host to be on if the host list is empty.